lib/wallaby/session.ex

```diff
@@ -44,7 +44,7 @@ defmodule Wallaby.Session do
     id: String.t,
     session_url: String.t,
     url: String.t,
-    server: pid(),
+    server: pid | nil,
```

Comment:
This is a super small nitpick but could we use an atom here like `:empty` or `:none` or maybe even `:selenium`? Just to signify that we don't actually use the server key for certain drivers.